queries.md

```diff
@@ -1,25 +1,85 @@
 # Database Queries
 
 ## find all customers that live in London. Returns 6 records.
+select * from customers where city is 'London'
+
 
 ## find all customers with postal code 1010. Returns 3 customers.
+select * from customers where PostalCode like '1010'
+
+or
+
+select * from customers where PostalCode is '1010'
+
 
 ## find the phone number for the supplier with the id 11. Should be (010) 9984510.
+select Phone from suppliers where SupplierID is 11
+
 
 ## list orders descending by the order date. The order with date 1997-02-12 should be at the top.
+select * 
+from orders 
+order by OrderDate
+desc
+
 
 ## find all suppliers who have names longer than 20 characters. You can use `length(SupplierName)` to get the length of the name. Returns 11 records.
+select * 
+from suppliers 
+where length(SupplierName) > 20
+
 
 ## find all customers that include the word "market" in the name. Should return 4 records.
+select * 
+from customers
+where CustomerName like '%market%'
+
 
 ## add a customer record for _"The Shire"_, the contact name is _"Bilbo Baggins"_ the address is _"1 Hobbit-Hole"_ in _"Bag End"_, postal code _"111"_ and the country is _"Middle Earth"_.
+insert into Customers (CustomerName, ContactName, Address, City, PostalCode, Country)
+values ('The Shire','Bilbo Baggins','1 Hobbit-Hole','Bag End','111','Middle Earth')
```

Comment:
perfect. an interesting thing with insert queries is that you can get away with not listing each individual column name like you do in the first line here, but then it checks against the total number of columns in the table and what you supply and would throw an error since there are 7 including customerId in the table while you only supply 6--you'd actually have to supply an explicit customerId for the new table entry instead of it being autogenerated like it is here. this way takes longer to write but is much better for that reason